docs/manual/java/guide/services/ServiceInfo.md

```diff
@@ -0,0 +1,30 @@
+# Service Metadata
+
+Service metadata, also referred to as `ServiceInfo`, includes a name and a collection of Service ACLs. Metadata is computed automatically in most scenarios and you won't need to review it or even provide it.
+
+## Service Info scenarios
+
+There are four possible scenarios supported by Lagom:
```

Comment:
I think I'd get rid of the "Service Info scenarios" header and this sentence and just let the list of four scenarios flow from the first paragraph.